### ISSUE TITLE
Fails to run on 6809 CPU

### ISSUE THREAD

**Issue by nick3092** [nowhereman999/CoCo3_MemTest2023#3]
On both real hardware and in MAME, this will not run on 6809 CPUs.  The experience is the same.  After loading, you are left with a broken screen that constantly seems to try and refresh.  It also seems to be detecting it as a 6309, based on the broken screen.

![Mem Test 2023 6809 MAME](https://github.com/nowhereman999/CoCo3_MemTest2023/assets/66426173/03916cac-8128-4351-8606-3fa5b6de6d14)

![Mem Test 2023 6809 HW](https://github.com/nowhereman999/CoCo3_MemTest2023/assets/66426173/ce27e516-7191-4454-92e3-8b047f350ded)

**Comment by nowhereman999** [comment]
Thanks for trying the memory tester, I just tried it in MAME using the .DSK file here as a 6809 and it works fine.  Make sure you are using the latest versions which is V1.21
[MemTestV1.216809.pdf](https://github.com/nowhereman999/CoCo3_MemTest2023/files/15213673/MemTestV1.216809.pdf)

**Comment by nick3092** [comment]
I used the only one listed under Releases here on github, which is 1.01.  Can you update the releases section for 1.21?

**Comment by nowhereman999** [comment]
Hi Nick3092, 
I'm not a real user of GitHub, it's basically somewhere I share my projects on the internet.  I don't even know what you mean when you say you see it listed under releases.
This is the most current version:
https://github.com/nowhereman999/CoCo3_MemTest2023

From the website just click on the Green Code button then select Download Zip
Then you'll have the latest version

If you can show me where the releases are and explain how I can update it I'll try and update it there too.
Cheers,
Glen

**Comment by nick3092** [comment]
OK, I'll try grabbing it that way.

Normally when a software package is ready/stable, the author publishes it in the Releases section vs users grabbing code from the repository.  I'm not familiar Enough to say how you do it, just that normally people would go here to look for "stable" releases.

![GH Releases](https://github.com/nowhereman999/CoCo3_MemTest2023/assets/66426173/51d3bdb9-c119-4b69-b004-8fed165877ca)

I did find a link explaining how you would add to the Releases section:

[https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository](url)

**Comment by nowhereman999** [comment]
Thanks for the help Nick, I tried to update the release to V1.21 but it kept asking me for stuff I don't know. So I decided rather than fighting with it I simply removed the release (I found the trash can icon easy enough) At least for now people will have to grab it the same way you just did.  Maybe when I get time I'll learn more about how GitHub works and have the release point to version 1.21.

Cheers,
Glen
